shell/browser/browser_win.cc

```diff
@@ -532,13 +600,42 @@ Browser::LoginItemSettings Browser::GetLoginItemSettings(
   base::win::RegKey key(HKEY_CURRENT_USER, keyPath.c_str(), KEY_ALL_ACCESS);
   base::string16 keyVal;
 
+  // keep old openAtLogin behaviour
   if (!FAILED(key.ReadValue(GetAppUserModelID(), &keyVal))) {
     base::string16 exe = options.path;
     if (FormatCommandLineString(&exe, options.args)) {
       settings.open_at_login = keyVal == exe;
     }
   }
 
+  std::vector<Browser::LaunchItem> launch_items;
+  // if called with no path, the executable_launch_at_login should be the same
+  // as open_at_login.
+  if (options.path.size() == 0) {
```

Comment:
The default value of `path` is `process.execPath` so shouldn't this never be empty?